models/Kobayashi_Dogleg/run_and_store.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+import openmc_fusion_benchmarks as ofb
+import helpers
+import numpy as np
+import h5py
+from pathlib import Path
+
+
+# ignore NaturalNameWarnings
+import warnings
+from tables import NaturalNameWarning
+warnings.filterwarnings('ignore', category=NaturalNameWarning)
+
+
+def _parse_args():
+    """Parse and return commandline arguments"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-t", "--when", type=str, default='n/a')
+    parser.add_argument("-w", "--where", type=str, default='n/a')
+    parser.add_argument("-x", "--xslib", type=str, default='mgxs')
+
+    args = parser.parse_args()
+
+    return args
+
+
+def main():
+    """Run the Kobayashi Dogleg simulation and store an hdf file in results_database/"""
+    args = _parse_args()
+    
+    # generate weight windows tallies
+    try:
+        p = subprocess.Popen(["python3", "openmc_model.py", "--Problem_I","--weight_windows"])
+    except:
+        p = subprocess.Popen(["python", "openmc_model.py", "--Problem_I","--weight_windows"])
+
+    # wait for the simulation to finish
+    p.wait()
+
+    # run simulation without scattering
+    try:
+        p = subprocess.Popen(["python3", "openmc_model.py", "--Problem_I"])
+    except:
+        p = subprocess.Popen(["python", "openmc_model.py", "--Problem_I"])
+
+    # wait for the simulation to finish
+    p.wait()
+
+    try:
+        p = subprocess.Popen(["python3", "openmc_model.py", "--weight_windows"])
+    except:
+        p = subprocess.Popen(["python", "openmc_model.py", "--weight_windows"])
+
+    # wait for the simulation to finish
+    p.wait()
+
+    # run simulation with scattering
+    try:
+        p = subprocess.Popen(["python3", "openmc_model.py"])
+    except:
+        p = subprocess.Popen(["python", "openmc_model.py"])
+
+    # wait for the simulation to finish
+    p.wait()
```

Comment:
I would not default to a ww generation run. Especially for the `run_and_store` file. I would rather provide already some ww files and leave the ww generation as optional. Or does this benchmark specifically focus on ww generation and performances?

```suggestion
    # run simulation without scattering
    try:
        p = subprocess.Popen(["python3", "openmc_model.py", "--Problem_I"])
    except:
        p = subprocess.Popen(["python", "openmc_model.py", "--Problem_I"])

    # wait for the simulation to finish
    p.wait()

    # run simulation with scattering
    try:
        p = subprocess.Popen(["python3", "openmc_model.py"])
    except:
        p = subprocess.Popen(["python", "openmc_model.py"])

    # wait for the simulation to finish
    p.wait()
```